src/modules/delete-completed.js

```diff
@@ -0,0 +1,15 @@
+import { DisplayList } from '../index';
+import { LocalStorage } from './local-storage';
+
+const clear = document.getElementById('clear-btn');
+
+function deleteCompleted() {clear.addEventListener('click', () => {
+  let tasks = LocalStorage.getTasks();
+  const newTasks = tasks.filter((task) => task.completed !== true);
+  console.log(newTasks);
```

Comment:
kindly. My recommendation is not to use a console.log. Remember to use best practices for coding.